src/server.rs

```diff
@@ -526,7 +514,10 @@ impl<C> SccacheService<C>
                           cwd: String,
                           env_vars: Vec<(OsString, OsString)>,
                           tx: mpsc::Sender<Result<Response>>) {
-        let cache_control = if self.force_recache {
+        let force_recache = env_vars.iter().any(|&(ref k, ref _v)| {
+            k.as_os_str() == OsStr::new("SCCACHE_RECACHE")
```

Comment:
`OsStr` has a `PartialEq` impl that handles comparing to '&str', so you don't need the `OsStr::new`, I don't think.